modules/v2x/v2x_proxy/obu_interface/grpc_interface/grpc_server.cc

```diff
@@ -136,6 +150,21 @@ void GrpcServerImpl::GetMsgFromGrpc(std::shared_ptr<ObuRsi> *ptr) {
   (*ptr)->CopyFrom(latest_rsi_);
 }
 
+void GrpcServerImpl::GetMsgFromGrpc(
+    std::shared_ptr<::apollo::v2x::V2XObstacles> *ptr) {
```

Comment:
since have used std::shared_ptr, why still use * ?

Reply:
```apollo.sh lint``` don't allow to use non-conf refs, also in order to be consistent with the existing code style.